shell/platform/windows/flutter_window_win32.cc

```diff
@@ -229,6 +229,22 @@ void FlutterWindowWin32::OnScroll(double delta_x,
                                       device_id);
 }
 
+void FlutterWindowWin32::OnSetFocus() {
+  binding_handler_delegate_->OnResumed();
+}
+
+void FlutterWindowWin32::OnKillFocus() {
+  binding_handler_delegate_->OnInactive();
+}
+
+void FlutterWindowWin32::OnMinimized() {
+  binding_handler_delegate_->OnPaused();
+}
+
+void FlutterWindowWin32::OnRestoredFromMinimized() {
+  binding_handler_delegate_->OnInactive();
```

Comment:
Is this right? Shouldn't restoring a window be similar to setting focus on the window?

Reply:
The OnRestoredFromMinimized() is  implicitly depended on which is is called before OnSetFocus() and WM_SIZE is called before WM_SETFOCUS, when the window is restored from minimized.

It can be broken when another app sends WM_SIZE so I'll re-think this.

Reply:
I added FlutterWindow::SendLifecycleEvent and removed this line. I'll mark this as resolved.